enarx-keep/src/backend/kvm/vm/x86_64.rs

```diff
@@ -15,6 +15,7 @@ pub struct VMSetup<'a> {
 }
 
 impl VMSetup<'_> {
+    #[allow(dead_code)]
```

Comment:
please don't ... let @connorkuehl decide

Reply:
Wouldn't dream of editing the `kvm` crate normally, but it was emitting a warning that was showing up in my `stderr` test. Maybe there is another way to keep from picking that up.

Reply:
How so? Is it recompiling everything?

Reply:
My opinion here is that we should just remove the dead code rather than `#[allow(dead_code)]` -- this function was superseded by one of Harald's PRs anyway so it's no longer required.

I'm also surprised to hear that the compilation process of the crate is polluting the integration tests stderr. I think that should be addressed directly.

Reply:
I was having a lot of trouble with extraneous cargo output in the `stderr` test generally, and this happened pretty late in the day, so between those two things it's likely that the error is on my end. I'll remove this change and see if it still crops up.